content/client-lib-development-guide/features.textile

```diff
@@ -86,6 +86,15 @@ h3(#restclient). RestClient
 ** @(RSC19c)@ The library will configure the @Accept@ and @Content-Type@ type headers to reflect whether the client is configured to use a binary or JSON based protocol (see "RSC8":#RSC8). All requests are encoded and decoded into Json or MsgPack as appropriate automatically by the library. Binary @body@ payloads are not supported at this time
 ** @(RSC19d)@ @request@ method returns an @HttpPaginatedResponse@ object that inherits from the @PaginatedResult@ object to provide details on the response plus paging support where applicable. See "HP1":#HP1 for more details
 ** @(RSC19e)@ If the HTTP network request fails for reasons such as a timeout (after the underlying fallback host attempts have failed where applicable, see "RSC15":#RSC15), then the @request@ method should indicate an error in an idiomatic way for the platform
+* @(RSC20)@ Unexpected internal library exception handling:
```

Comment:
Ok, so this is not relevant as far as I can tell to REST as it's effectively stateless, hence I added [`RTC11 `](https://github.com/ably/docs/pull/293/commits/51cf0c357b274ce9058188c9d85adc41d73a59c7#diff-11900b395df266bc2bbfc1d11bdfc7a0R296). I am happy to revise `RTC11` though to accommodate this feedback.